frontend/src/api/Room.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios';
+import { ErrorResponse, errorHandler } from './Error';
+
+export type Room = {
+  message: string,
+  roomId: string;
+};
+
+const basePath = '/api/v1/rooms';
+const routes = {
+  createRoom: `${basePath}/`,
+  joinRoom: `${basePath}/`,
+};
+
+export const createRoom = (): Promise<Room | ErrorResponse> => axios.post<Room>(routes.createRoom)
```

Comment:
Should we be using `axios` instead of `fetch(...)` in the `Problem.ts` file? I know you didn't change that here, but it may be good to note to change that in the future if we want to maintain consistency.

Reply:
Pushed a commit to change that since it should be a pretty simple fix.

Reply:
Awesome, thank you!